parser.y

```diff
@@ -422,6 +422,7 @@ import (
 	nulls		"NULLS"
 	offset		"OFFSET"
 	only		"ONLY"
+	opttrace        "OPT_TRACE"
```

Comment:
To pass the consistency check, we can add `opttrace` to the tokenMap in `misc.go`.